scripts/valhalla_build_config

```diff
@@ -477,9 +481,9 @@ def add_leaf_args(path, tree, leaves, parser):
     h = help_text
     for k in keys:
       h = h[k]
-    t = comma_separated_list if type(tree) == type([]) else type(tree)
+    t = comma_separated_list if type(tree) == type([]) else (type(tree) if type(tree) != type(optional()) else tree.type)
```

Comment:
here we get the type we want to accept for this arg in argparse. if the type is optional we bubble out the internal type to argparse